src/anm/interop/playerjs-io.js

```diff
@@ -0,0 +1,190 @@
+var engine = require('engine'),
+    C = require('../constants.js');
+
+var playerJsVersion = '0.0.10';
+
+var adapter = {
+    play: function() {
+        this.play();
+    },
+
+    pause: function() {
+        this.pause();
+    },
+
+    getPaused: function() {
+        return this.state.happens === C.PAUSED;
+    },
+
+    mute: function() {
+        this.mute();
+    },
+
+    unmute: function() {
+        this.unmute();
+    },
+
+    isMuted: function() {
+        return this.muted;
+    },
+
+    setVolume: function(message) {
+        var volume = message.value;
+        player.volume(volume/100);
+    },
+
+    getVolume: function() {
+        return player.volume()*100;
+    },
+
+    getDuration: function() {
+        return player.state.duration;
+    },
+
+    setCurrentTime: function(message) {
+        var time = message.value;
+        this.pause().play(time);
+    },
+
+    getCurrentTime: function() {
+        return this.state.time;
+    },
+
+    setLoop: function(message) {
+        var loop = message.value;
+        this.repeat = loop;
+    },
+
+    getLoop: function() {
+        return this.repeat;
+    },
+
+    addEventListener: function(message) {
+        var event = message.value,
```

Comment:
Is it possible to use `provideEvents` from `events.js` here instead of custom Event System?